easybuild/easyconfigs/c/CPPE/CPPE-0.3.3a0-GCC-12.3.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'CMakeMake'
+
+name = 'CPPE'
+version = '0.3.3a0'
```

Comment:
@PetrKralCZ This is a pre-release, I'm not sure it's wise to use this already...

Reply:
I also have a `0.3.1` version ready. Should I open PR for that one?